.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "trailingComma": "none",
+  "singleQuote": true,
+  "printWidth": 80,
+  "tabWidth": 4
```

Comment:
Tak samo do dyskusji ale jak dla mnie 2 jest wystarczające

Reply:
I would reject. I think for json/xml files 2 is sufficient, but for code is quite unreadable.